packages/modules/viewer-react/src/tests/components/app-ui/widgets/BasicToolWidget.test.tsx

```diff
@@ -197,13 +184,8 @@ describe("BasicToolWidget", () => {
       10,
       CoreTools.selectElementCommand
     );
-    // TODO 3.0
```

Comment:
we should be able to add these back now that you added measure tools I think?

Reply:
Will add PBI to add tests for ui Providers later